pycbc/workflow/core.py

```diff
@@ -1517,8 +1518,7 @@ def to_segment_xml(self, override_file_if_exists=False):
         # create XML doc and add process table
         outdoc = ligolw.Document()
         outdoc.appendChild(ligolw.LIGO_LW())
-        process = ligolw_utils.process.register_to_xmldoc(outdoc, sys.argv[0],
-                                                          {})
+        process = ligolw_process.register_to_xmldoc(outdoc, sys.argv[0], {})
```

Comment:
I'm installing from the pycbc-glue repo. I might have to blow away my install and go from scratch, as currently I get:
```
>>> from glue.ligolw import table, lsctables, ligolw
>>> from glue.ligolw import utils as ligolw_utils
>>> ligolw_utils.__file__
'/home/c1239174/pycbc-dev/lib/python2.6/site-packages/glue/ligolw/utils/__init__.pyc'
>>> import ligolw_utils.process
Traceback (most recent call last):
  File "<ipython-input-4-3f945f03a313>", line 1, in <module>
    import ligolw_utils.process
ImportError: No module named ligolw_utils.process

>>> import glue.ligolw.utils.process
>>> glue.ligolw.utils.process.__file__
'/home/c1239174/pycbc-dev/lib/python2.6/site-packages/glue/ligolw/utils/process.pyc'
```